web/translation/translate.en_US.toml

```diff
@@ -318,9 +318,9 @@
 "FreedomStrategyDesc" = "Set the output strategy of the network in the Freedom Protocol."
 "RoutingStrategy" = "Configure Domains Routing Strategy"
 "RoutingStrategyDesc" = "Set the overall routing strategy for DNS resolving."
-"Torrent" = "Ban BitTorrent Usage"
-"TorrentDesc" = "Change the configuration template to avoid using BitTorrent by users."
-"PrivateIp" = "Ban Private IP Ranges to Connect"
+"Torrent" = "Block BitTorrent Usage"
+"TorrentDesc" = "Change the configuration template to avoid using BitTorrent by clients."
+"PrivateIp" = "Disable Connection to Private IPs"
```

Comment:
It is not about disable a connection. It is exactly ban these ips to be able to communicate.